ci-operator/step-registry/openshift/upgrade/aws-check-dev-symlinks/openshift-upgrade-aws-check-dev-symlinks-workflow.metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+	"path": "openshift/upgrade/aws-check-dev-symlinks/openshift-upgrade-aws-check-dev-symlinks-workflow.yaml",
```

Comment:
There already is directory `openshift/upgrade/aws`, so I would put it there. In addition, I would add `storage` to the workflow name, so it's obvious what it tests and who owns it, + shorten it `ci-operator/step-registry/openshift/upgrade/aws/storage-symlinks/`? `storage-udev`?